wfl/cli/commands/eval.py

```diff
@@ -0,0 +1,81 @@
+import click
+
+from quippy.potential import Potential
+
+from wfl.autoparallelize.autoparainfo import AutoparaInfo
+from wfl.cli import cli_options as opt
+
+
+@click.command("gap")
+@click.option("--gap-fname", '-g', type=click.Path(), help="Path to GAP XML")
```

Comment:
I think I'd prefer for the parameter file name argument to be the same for all the potentials.  In fact, all of these options (except the extra MACE ones) can and should be standardized by centralizing them for all the ML evaluation subcommands.

Reply:
done